Formula/g/graphviz.rb

```diff
@@ -1,12 +1,19 @@
 class Graphviz < Formula
   desc "Graph visualization software from AT&T and Bell Labs"
   homepage "https://graphviz.org/"
-  url "https://gitlab.com/graphviz/graphviz.git",
-      tag:      "12.0.0",
-      revision: "a2902960366a3df263186bf099a7bfe2334fd1fe"
   license "EPL-1.0"
   version_scheme 1
-  head "https://gitlab.com/graphviz/graphviz.git", branch: "main"
+
+  stable do
+    url "https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/12.0.0/graphviz-12.0.0.tar.xz"
+    sha256 "720d5392ccab391d50ac2131048705285fbd3fe8948cdb27c7e7b56160d11a9f"
```

Comment:
Have you checked that it generates version information correctly? IIRC I tried this before but dropped it because it used `git` to look up tags.

Reply:
Do you remember where exactly?

Basic checks seem to be correct:
```console
❯ dot --version
dot - graphviz version 12.0.0 (20240704.0754)

❯ cat $(brew --prefix graphviz)/include/graphviz/graphviz_version.h
#pragma once
#define GVPLUGIN_CONFIG_FILE "config6"
#define GVPLUGIN_VERSION 6
#define PACKAGE_BUGREPORT "https://gitlab.com/graphviz/graphviz/-/issues"
#define PACKAGE_NAME "graphviz"
#define PACKAGE_STRING "graphviz 12.0.0"
#define PACKAGE_TARNAME "graphviz"
#define PACKAGE_URL ""
#define PACKAGE_VERSION "12.0.0"
```

Reply:
Huh, maybe they've fixed that. IIRC `configure` used to check `git`. (Or maybe that was `autogen.sh`, and I was just confused.)